project/BuildHelper.scala

```diff
@@ -161,15 +160,18 @@ object BuildHelper {
   def stdSettings(prjName: String) = Seq(
     name := s"$prjName",
     scalacOptions := stdOptions,
-    crossScalaVersions := Seq("2.12.9", "2.13.0", "2.11.12"),
+    crossScalaVersions := Seq("2.12.10", "2.13.1", "2.11.12"),
     scalaVersion in ThisBuild := crossScalaVersions.value.head,
     scalacOptions := stdOptions ++ extraOptions(scalaVersion.value, optimize = !isSnapshot.value),
-    libraryDependencies ++= compileOnlyDeps ++ testDeps,
+    libraryDependencies ++= testDeps,
     libraryDependencies ++= {
       if (isDotty.value)
-        Seq()
+        Seq("com.github.ghik" % "silencer-lib_2.13.1" % "1.4.4" % Provided)
```

Comment:
@ghostdogpr are we happy with this hack?

Reply:
No other way I guess? I’d say it’s fine since this plugin has very limited impact.